instrumentation/jetty/jetty-8.0/javaagent/src/test/groovy/JettyContinuationHandlerTest.groovy

```diff
@@ -1,67 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-import org.eclipse.jetty.continuation.Continuation
-import org.eclipse.jetty.continuation.ContinuationSupport
-import org.eclipse.jetty.server.Request
-import org.eclipse.jetty.server.handler.AbstractHandler
-
-import javax.servlet.ServletException
-import javax.servlet.http.HttpServletRequest
-import javax.servlet.http.HttpServletResponse
-import java.util.concurrent.ExecutorService
-import java.util.concurrent.Executors
-
-// FIXME: We don't currently handle jetty continuations properly (at all).
-abstract class JettyContinuationHandlerTest extends JettyHandlerTest {
```

Comment:
Jetty `Continuation` was deprecated in jetty 9.4 and removed in jetty 10, so this test doesn't compile against jetty 10 (see latestDepTestLibrary bump in the gradle file).

Since this test doesn't run anyways (it's abstract) and based on the FIXME we don't support Continuation anyways, it seems ok to just delete it.